### DISCUSSION TITLE
Support formatting all files supported by prettier

### DISCUSSION THREAD

**Discussion by Goldziher** [biomejs/biome#3197] [Ideas]
The suggestion / request is to extend the formatter to have full parity with prettier formatting of files. For example, prettier formats yaml. 

Why? Because many projects use prettier to format these files, abd even use plugins to format stuff such as XML, sort JSON, format scss etc. 

I wanted to migrate to biome formatting but this forces me to use both tools.

**Comment by arendjr** [comment]
We provide an overview of language support on this page: https://biomejs.dev/internals/language-support/

In addition, current developments, including work on supporting more languages, is tracked here: https://github.com/biomejs/biome/issues/2455

Feel free to open individual discussions for languages you feel are missing, and for which no discussion exists yet (for instance, YAML support also has a thread here: https://github.com/biomejs/biome/discussions/199). I will close this one so we can keep discussion more focused.
